sway/tree/view.c

```diff
@@ -828,6 +834,10 @@ void view_map(struct sway_view *view, struct wlr_surface *wlr_surface,
 		input_manager_set_focus(&view->container->node);
 	}
 
+	foreign_toplevel_state.app_id = (char *)view_get_app_id(view);
+	foreign_toplevel_state.title = (char *)view_get_class(view);
```

Comment:
Is this ok, or should the wlroots struct use the const qualifier?

Reply:
IMHO would be better if the wlroots struct used the const qualifier

Reply:
Done